### PR TITLE
Add OmegaT 5.7.1; Fix #94592

### LINKED ISSUE #94592: [Package Request]: OmegaT
### Package Requested

- [ ] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a script file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: The OmegaT project
Publisher Url: https://omegat.org
Package Name: OmegaT
Description: The free computer aided translation (CAT) tool for professionals
Package Version: 5.7.1
Installer URL:   InstallerUrl: https://sourceforge.net/projects/omegat/files/OmegaT%20-%20Latest/OmegaT%205.7.1/OmegaT_5.7.1_Beta_Windows_64_Signed.exe/download

Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by ktgw0316**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94593)

### CODE REVIEW

**Comment by ktgw0316** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Validation appears to be impacted by a slow download. I'm guessing that the download duration is exceeding the installation timeout, and that this is being reflected by different errors. (These are also clouded by an unrelated issue.) 

In a sandbox with manual timing, the download was getting about 2 MB/min or about 33.3 KB/s. For this 168 MB package, this would take 84 minutes.

File: manifests/o/OmegaT/OmegaT/5.7.1/OmegaT.OmegaT.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: OmegaT.OmegaT
+PackageVersion: 5.7.1
+PackageLocale: en-US
+Publisher: OmegaT
+PackageName: OmegaT
```
Comment (stephengillie) [inline manifests/o/OmegaT/OmegaT/5.7.1/OmegaT.OmegaT.locale.en-US.yaml:7]:
```suggestion
PublisherUrl: https://omegat.org/
PublisherSupportUrl: https://omegat.org/support
PackageName: OmegaT
PackageUrl: https://omegat.org/download
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**